main.cpp

```diff
@@ -84,13 +84,16 @@ struct Transform
         
 int main()
 {
-    T varA( 10.0f, "A" );                                             //6
-    T varB( 100.0f, "B" );                                             //6
+    const T varA( 100.0f, "A" );                                             //6
+    const T varB( 10.0f, "B" );                                             //6
```

Comment:
You didn't need to change this.

Reply:
removed const object usage outside of const requesting func ( new commit )